src/libraries/System.Net.NetworkInformation/tests/FunctionalTests/NetworkInterfaceBasicTest.cs

```diff
@@ -21,6 +21,7 @@ public NetworkInterfaceBasicTest(ITestOutputHelper output)
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)] // throws System.IO.DirectoryNotFoundException : Could not find a part of the path '/etc/resolv.conf'
```

Comment:
Is this an acceptable experience, getting a DirectoryNotFoundException for /etc/resolv.conf ?

Reply:
note that /etc/resolv.conf is generally present but not required for name resolution to work on Unix. 
I'm wondering if it would make sense to fix either the test or production code.

Reply:
Do I need to file an issue?